docs/reference_guides/model_libraries/generic/unit_models/shell_and_tube_1d.rst

```diff
@@ -0,0 +1,139 @@
+* property_package_args - set of arguments to be passed to the shell side Property Blocks when they are created.
+* transformation_method - argument to specify the DAE transformation method for the shell side; should be compatible with the Pyomo DAE TransformationFactory
+* transformation_scheme - argument to specify the scheme to use for the selected DAE transformation method; should be compatible with the Pyomo DAE TransformationFactory
+
+Additionally, Shell and Tube Heat Exchanger units have the following construction arguments for each side which are passed
+to the ControlVolume1DBlocks for determining which terms to construct in the balance equations for the
+hot and cold sides.
+
+========================= =================
+Argument                  Default Value
+========================= =================
+dynamic                   useDefault
+has_holdup                False
+material_balance_type     'componentTotal'
+energy_balance_type       'enthalpyTotal'
+momentum_balance_type     'pressureTotal'
+has_phase_equilibrium     False
+has_heat_transfer         True
+has_pressure_change       False
+========================= =================
+
+Variables
+---------
+
+1-D Shell and Tube Heat Exchanger units add the following additional Variables beyond those created by the
+ControlVolume1DBlock Block.
+
+======================== ==================================== ==================================
+Variable                 Name                                 Notes
+======================== ==================================== ==================================
+:math:`L`                length                               Reference to hot_side.length
+:math:`D_{shell}`        shell_diameter                       Diameter of shell
+:math:`D_{tube, inner}`  tube_inner_diameter                  Inner diameter of tubes
+:math:`D_{tube, outer}`  tube_outer_diameter                  Outer diameter of tubes
+:math:`N_{tubes}`        number_of_tubes
+:math:`T_{wall}`         temperature_wall                     Temperature of tube wall material
+:math:`U_{hot,t,x}`      hot_side_heat_transfer_coefficient
+:math:`U_{cold,t,x}`     cold_side_heat_transfer_coefficient
+======================== ==================================== ==================================
+
+Additional Constraints
+----------------------
+
+1-D Shell and TubeHeat Exchanger models write the following additional Constraints to describe the heat transfer between the
+two sides of the heat exchanger.
+
+Firstly, heat transfer for the hot and cold sides to the tube wall is calculated as:
+
+.. math:: Q_{hot,t,x} = -U_{hot_t,x} \times N_{tubes} \times \pi \times D_{tube, outer} \times (T_{hot,t,x}-T_{wall,t,x}))
+
+.. math:: Q_{cold,t,x} = U_{cold_t,x} \times N_{tubes} \times \pi \times D_{tube, inner} \times (T_{wall,t,x}-T_{cold,t,x}))
```

Comment:
Q is like a heat flux but instead of pre area it's per length.  Should it have delta x in there?

Reply:
Probably not a delta x (that is the equation that goes into the model), but maybe a comment to clarify that that it is per unit length.